test/over_react/util/react_wrappers_test.dart

```diff
@@ -574,6 +576,264 @@ main() {
         }));
       });
 
+      group('traverses children of Wrapper components', () {
+        group('and returns props for a', () {
+          group('composite JS ReactComponent', () {
+            test('', () {
+              ReactComponent renderedInstance = render(OneLevelWrapper()(
+                testJsComponentFactory({
+                  'jsProp': 'js',
+                  'style': testStyle,
+                }, testChildren)
+              ));
+
+              expect(getProps(renderedInstance, traverseWrappers: true), equals({
+                'jsProp': 'js',
+                'style': testStyle,
+                'children': testChildren
+              }));
+            });
+
+            test('even when there are multiple levels of wrappers', () {
+              ReactComponent renderedInstance = render(TwoLevelWrapper()(
+                OneLevelWrapper()(
+                  testJsComponentFactory({
+                    'jsProp': 'js',
+                    'style': testStyle,
+                  }, testChildren)
+                )
+              ));
+
+              expect(getProps(renderedInstance, traverseWrappers: true), equals({
+                'jsProp': 'js',
+                'style': testStyle,
+                'children': testChildren
+              }));
+            });
+
+            test('even when props change', () {
+              var mountNode = new DivElement();
+              ReactComponent renderedInstance = react_dom.render(OneLevelWrapper()(
+                testJsComponentFactory({
+                  'jsProp': 'js',
+                  'style': testStyle,
+                }, testChildren)
+              ), mountNode);
+
+              expect(getProps(renderedInstance, traverseWrappers: true), equals({
+                'jsProp': 'js',
+                'style': testStyle,
+                'children': testChildren
+              }));
+
+              renderedInstance = react_dom.render(OneLevelWrapper()(
+                testJsComponentFactory({
+                  'jsProp': 'other js',
+                  'style': testStyle,
+                }, testChildren)
+              ), mountNode);
+
+              expect(getProps(renderedInstance, traverseWrappers: true), equals({
+                'jsProp': 'other js',
+                'style': testStyle,
+                'children': testChildren
+              }));
+            });
+
+            test('except when traverseWrappers is false', () {
+              ReactComponent renderedInstance = render(OneLevelWrapper()(
+                testJsComponentFactory({
+                  'jsProp': 'js',
+                  'style': testStyle,
+                }, testChildren)
+              ));
+
+              expect(getProps(renderedInstance), isNot({
```

Comment:
Would probably be better to verify explicitly that the `OneLevelWrapper`'s props are returned.